### ISSUE TITLE
Telegram cause bot to crash

### ISSUE THREAD

**Issue by ppa993** [NECROBOTIO/NecroBot#4557]
### Expected Behavior
Telegram bot show list of available commands

### Actual Behavior
Necrobot crash, wont be able to restart unless disable telegram

### Your config.json
enable telegram

### Steps to Reproduce
Send "/top 5 iv" command

**Comment by kkekof** [comment]
/top iv 5 need to send :)

**Comment by ppa993** [comment]
btw, any how to enable telegram again without setting up another bot ?

**Comment by theloser** [comment]
@ppa993 Have you lost your API key? Then just use "BotFather" to /deletebot and re-create a new one.

**Comment by WoShiSaBi** [comment]
for your api key like so
APIKEY: "SDJAIOSDJAIOSJDIOASJD" <--- example 
rem to put "

**Comment by ppa993** [comment]
@theloser no, it's work fine until I send command in wrong format. There is nothing to do with API key

**Comment by theloser** [comment]
I still don't get it: yes, you can crash the bot if you send commands in wrong format (sometimes the bot crashes, sometimes it just ignores them, weired). But You don't have to setup a new (telegram)bot afterwards, you just restart it and it reconnects to telegram -- am I missing something?

**Comment by ppa993** [comment]
After restart, it's still trying to respone to the last command, which is in wrong format.

**Comment by theloser** [comment]
Wow, really? I didn't realize that. My bot just crashed, I restarted it, sent the correct command through telegram and everything works fine.

**Comment by ChemiHatake** [comment]
Ill make it just send a correct command template if it was wrong instead of crashing

**Comment by ppa993** [comment]
@ChemiHatake thank a lots

**Comment by ChemiHatake** [comment]
@ppa993 fixed, will be submitting my pr later
